lib/hsa/mcwamp_hsa.cpp

```diff
@@ -105,6 +106,31 @@ unsigned HCC_DB = 0;
 int HCC_MAX_QUEUES = 20;
 
 
+static std::ofstream g_hccProfileFile; // if using a file open it here
+static std::ostream *g_hccProfileStream = nullptr; // point at file or default stream
+
+#define HCC_PROFILE_VERBOSE_BASIC                   (1 << 0)
+#define HCC_PROFILE_VERBOSE_TIMESTAMP               (1 << 1)
+#define HCC_PROFILE_VERBOSE_BARRIER                 (1 << 2)
+
+
+int HCC_PROFILE=0;
+int HCC_PROFILE_VERBOSE=0;
+char * HCC_PROFILE_FILE=nullptr;
```

Comment:
@bensander i'm seeing more file-scope global variables growing... how about we move them into `HSAContext`?